Package.swift

```diff
@@ -275,7 +274,7 @@ var package = Package(
     ),
     .testTarget(name: "StorageTests", dependencies: ["Storage", "BraveSharedTestUtils"], resources: [.copy("fixtures/v33.db"), .copy("testcert1.pem"), .copy("testcert2.pem")]),
     .testTarget(name: "DataTests", dependencies: ["Data", "DataTestsUtils"]),
-    .testTarget(name: "SPMLibrariesTests", dependencies: ["GCDWebServers"]),
+    .testTarget(name: "SPMLibrariesTests", dependencies: []),
```

Comment:
This was the only thing in this test target so it needs to be removed entirely

Reply:
Removed.